frontend/src/containers/Subscription/SubscriptionFeed.js

```diff
@@ -20,6 +59,77 @@ class SubscriptionFeed extends Component {
     //     }
     // }
 
+    clickMoreButton = () => {
+        if (!this.props.subscriptionFinished) {
+            this.props.onGetSubscriptions({
+                page_number: this.props.subscriptionPageNum + 1,
+            })
+                .then(() => {
+                    const { subscriptions } = this.state;
+                    this.setState({
+                        subscriptions: subscriptions.concat(this.props.subscriptionItems),
+                    });
+                    this.clickMoreButtonNext();
+                }).catch(() => {});
+        } else {
+            this.clickMoreButtonNext();
+        }
+    }
+
+    clickMoreButtonNext = () => {
+        if (!this.props.recommendationFinished) {
+            this.props.onGetRecommendations({
+                page_number: this.props.recommendationPageNum + 1,
+            })
+                .then(() => {
+                    const { recommendations } = this.state;
+                    this.setState({
+                        recommendations: recommendations.concat(this.props.recommendationItems),
+                    });
+                    this.addRecoToSub();
+                }).catch(() => {});
+        } else {
+            this.addRecoToSub();
+        }
+    }
+
+    addRecoToSub = () => {
+        for (let i = this.state.recommendations.length - 1; i > this.state.start; i -= 1) {
+            const j = Math.floor(Math.random() * (i + 1));
+            [this.state.recommendations[i], this.state.recommendations[j]] = [
+                this.state.recommendations[j],
+                this.state.recommendations[i],
+            ];
+        }
+        let temp = [];
+        if (this.state.recommendations.length >= this.state.recoNum) {
+            temp = this.state.recommendations.splice(0, this.state.recoNum);
+        } else {
+            temp = this.state.recommendations.splice(0, this.state.recommendations.length);
+        }
+
+        for (let i = 0; i < temp.length; i += 1) {
+            const index = Math.random()
+            * (this.state.subscriptions.length - this.state.start)
+            + this.state.start;
+            this.state.subscriptions.splice(index, 0, temp[i]);
+        }
+        this.setState((prevState) => ({
```

Comment:
[Not important] Why did you use `prevState`?

Reply:
Because I need length of this.state.subscriptions and eslint wants me to use callback like thatㅠ

Reply:
OK. Well, I think this is better than my habit. I used to assign a variable of the previous state and use it in `setState`.